config/manager/manager.yaml

```diff
@@ -35,9 +35,14 @@ spec:
         - windows-machine-config-operator
         args:
         - "--debugLogging"
+        - "--metrics-bind-address=0.0.0.0:8443"
         image: controller:latest
         name: manager
         imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8443
```

Comment:
Given that the port is hard-coded in the manifest; What is the use case of the proposed flag `metrics-bind-address` ?

Reply:
The flag is already available through controller-runtime's server package, the intention is to avoid hardcoding in code and pass it through the CSV.